base/docs/basedocs.jl

```diff
@@ -1492,4 +1492,8 @@ See the manual section on [Tuple Types](@ref).
 """
 Tuple
 
+@doc """
+The standard library of Julia.
+""" -> Base
```

Comment:
As there is/will be a  "standard library", see https://github.com/JuliaLang/julia/tree/master/stdlib, it Base should not be explained as standard library.

Reply:
Good point. At the moment `?Base` does not anyway. Any idea how to fix that?

Reply:
You could document it as keyword
```
 @doc """
 bla bla
 """ -> kw"Base"
```